github-actions/verify-pr-title/src/common.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * This regex portion matches @W-0000@ through @W-999999999@,
+ * to enforce that the title contains a work record number consumable
+ * by Git2Gus.
+ * All pull request titles will require this segment.
+ */
+export const WORK_ITEM_PORTION = "@W-\\d{4,9}@"
+
+/**
+ * This regex matches one or more whitespace, comma, period, n-dash,
+ * colon, or semicolon characters. All pull request titles allow these
+ * characters as separators between portions
+ */
+export const SEPARATOR = "[\\s-.,;:]+";
```

Comment:
Do we want "\" or do we just want a space " ". I don't think we want tab or newline characters do we?

Reply:
Yeah, I'm fine restricting it to space if we want.